mobile/library/common/extensions/cert_validator/platform_bridge/platform_bridge_cert_validator.h

```diff
@@ -55,6 +56,11 @@ class PlatformBridgeCertValidator : public CertValidator, Logger::Loggable<Logge
   }
 
 private:
+  GTEST_FRIEND_CLASS(PlatformBridgeCertValidatorTest, ThreadCreationFailed);
```

Comment:
Why does the test need to be declared friend?

Reply:
It needs to call the private constructor, so that we can use a a mock `PosixThreadFactory`.